.github/workflows/scripts/check-bazel-cache-dir-size.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Copyright 2023 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script should be run from $MAGMA_ROOT
+
+# Relative path to cache directory from $MAGMA_ROOT
+RELATIVE_CACHE_DIR=$1
```

Comment:
Nit: Can we quote the inputs here and below?

Reply:
done